mne/images/README.rst

```diff
@@ -0,0 +1,37 @@
+.. -*- mode: rst -*-
+
+
+Documentation
+=============
+
+The images are used in `mne/viz/_brain` as icons in the toolbar of `_TimeViewer`.
```

Comment:
reStructuredText needs double backticks (2 on each end) to trigger code formatting.  Currently "mne/viz/_brain" and "_TimeViewer" will be in italics. Relevant throughout this file.